src/commands/deleteCosmosDBAccount.ts

```diff
@@ -0,0 +1,26 @@
+import * as vscode from 'vscode';
+import CosmosDBManagementClient = require("azure-arm-cosmosdb");
+import { IAzureNode } from 'vscode-azureextensionui';
+import { DialogBoxResponses } from '../constants';
+import { azureUtils } from '../utils/azureUtils';
+import * as util from '../utils/vscodeUtils';
+import { UserCancelledError } from 'vscode-azureextensionui';
+
+export async function deleteCosmosDBAccount(node: IAzureNode): Promise<void> {
+    const message: string = `Are you sure you want to delete account '${node.treeItem.label}'?`;
+    const result = await vscode.window.showWarningMessage(message, DialogBoxResponses.Yes, DialogBoxResponses.Cancel);
+    if (result === DialogBoxResponses.Yes) {
+        const docDBClient = new CosmosDBManagementClient(node.credentials, node.subscription.subscriptionId);
+        const resourceGroup: string = azureUtils.getResourceGroupFromId(node.treeItem.id);
+        const label: string = node.treeItem.label;
+        const accountName: string = label.substring(0, label.indexOf(" "));
```

Comment:
I would rather get the accountName from node.treeItem.id than node.treeItem.label (The id should never change, but we might change how the label is displayed - also it could be different formatting for different languages)

Take a look at `azureUtils.getResourceGroupFromId`. You can do something similar to get the accountName